reg/drone/service/actuator/util/Feedback.0.1.uavcan

```diff
@@ -0,0 +1,26 @@
+# This high-rate feedback should be published once immediately after a setpoint is applied.
+# It follows that the publication rate of these messages equals that of the setpoint messages.
+# When setpoint messages are not being emitted, the publication rate is implementation-defined.
+# The priority of this message should be the same as that of the corresponding setpoint message.
+
+float32 SETPOINT_EXPIRATION_TIMEOUT = 1.0  # [seconds]
```

Comment:
this seems high

Reply:
I forgot to mention in the docs that implementations are allowed to pick a lower timeout value, but not a higher one. Fixed now.

This value should not overconstrain low-rate actuators driving a high-time-constant mechanism where a low update rate may be acceptable.